webpack/assets/javascripts/react_app/pages/common/PageLayout/PageLayout.js

```diff
@@ -61,7 +66,11 @@ const PageLayout = ({
             )}
           </Col>
           <Col id="title_action" md={searchable ? 6 : 8}>
-            <div className="btn-toolbar pull-right">{toolbarButtons}</div>
+            <div className="btn-toolbar pull-right">
+              <Slot id="layout-toolbar-button" />
```

Comment:
@amirfefer will it make more sense to do:
```js
<Slot id="layout-toolbar-button" multi>
  {isExport && <ExportButton />}
  {isDocs && <DocumentationButton documentationURL={documentationURL} />}
</Slot>
```

Will it give more options for overriding in this case?

Reply:
It will cause the fill to replace the export/docs buttons

Reply:
It make sense @sharvit, but a change in the implementation is required.
meanwhile, we can do something like:

```js
<Slot id="layout-toolbar-button" multi />
<Fill id="exportButton" slotId=""layout-toolbar-button" weight={100}>
  {isExport && <ExportButton />}
</ Fill>
```